content/en/docs/reference/kubernetes-api/ref/authentication/serviceaccount-v1.md

```diff
@@ -0,0 +1,319 @@
+---
+weight: 1
+title: "ServiceAccount"
+description: "ServiceAccount binds together: * a name, understood by users, and perhaps by peripheral systems, for an identity * a principal that can be authenticated and authorized * a set of secrets."
```

Comment:
Is this a Markdown list on line 4?

Reply:
I think it's taken from the Go source code, so it's only kindaMarkdown™.

What could help: preserve the newlines, both in the front matter (multiline YAML value) and when it's repeated.

I have an idea to use a heuristic to decide whether to treat it as likely to be Markdown or whether to wrap it in <tt>```</tt> delimiters. That can arrive in a later iteration.

Reply:
I use the first phrase of the description of the definition extracted from the OpenAPI spec. I think the author wanted to represent a list here, yes

Reply:
Yes, I realize that this content is a list. You may want to think about some of the formatting issues now and
the incoming content that has to be displayed.

Reply:
I'd like to see the left hand nav tree help with the page navigation. Currently, not much happens there.
This make take some changes to get it right.
Previously mentioned that you may want to create `shortcodes` or `partial templates` to render different parts of the page.

Reply:
Some ideas about navigating the operations:

1. Use the left nav to assist in navigating the individual ops for a resource.
    - Break out the operations into individual pages or snippets, separate from the main resource page. This would grow the left nav tree which is already largish. Add these pages under the resource/operations directory. When navigating the APIs,  you may want a macro view of the APIs (block out the other content categories in the site), but still have the option
of navigating out of the reference section. Perhaps you want to the option to collapse or de-emphasize the non-reference part of the left tree to control the size of the left nav.
    - Customize the left hand nav to insert buttons for the operations list items (sub tree of operations under a resource/operations path). I can click on a "GET" button in the tree and see the API operation for that resource.
1.  Leave the operations as part of a single page resource, but add collapsible buttons with H2 level headings to represent
the individual ops. The operations appear similar to the swagger style of listings (GET, POST, ...).
Use the right hand tree to navigate the different operation sections. Select the buttons on the page to open/close the content. You lose the big picture view. For example, comparing a "GET" in the Pod sub tree with a "GET" in the Job sub tree in the left nav. You can only see the operations isolated per API.
This way does not add additional items to the left nav tree.

Could add a custom shortcode to handle the interactive examples on the page.

Adding any related metadata to the page provides an opportunity to reuse the text strings on the site (and localize strings).

Could collapse the included definitions into the main resource instead of listing after the resource definition.